src/authenticate.py

```diff
@@ -83,19 +83,21 @@ def exchange_cognito_code(event: dict, cognito_code: str) -> dict:
             client_id=client_id,
         )
     except requests.exceptions.RequestException as e:
-        structlog.get_logger().msg("Connection error to Cognito", exception=e)
+        logger.exception({"message": "Connection error to Cognito", "exception": e})
         raise InternalServerError()
     except jwt.InvalidTokenError as e:
-        structlog.get_logger().msg("id_token invalid", exception=e)
+        logger.exception({"message": "id_token invalid", "exception": e})
         raise InternalServerError()
 
-    structlog.get_logger().msg("Cognito ID token is valid")
+    logger.info("Cognito ID token is valid")
 
     return cognito_id_token
 
 
-def handler(event, context) -> dict:
-    del context  # unused
+@logger.inject_lambda_context
```

Comment:
Als deze door API gateway wordt aangeroepen:

```
from aws_lambda_powertools.logging import correlation_paths

@logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST)
```

Ik dacht dat dit door Lambda@Edge werd aangeroepen, maar is `request_ip = event['requestContext']['identity']['sourceIp']` dan niet verkeerd?

Reply:
daar kreeg ik allesinds een telenet IP te zien bij mijn beperkte test op sandbox.